web-app/src/components/CreateContract/CreateContractForms/dates/index.jsx

```diff
@@ -33,19 +32,18 @@ export const DateForm = props => {
           rules={{}}
           value={formData.startDate}
         />
-        <Hide hide={formData.isOpenEnded}>
-          <Input
-            label="End Date:"
-            type="date"
-            htmlAttrs={{
-              type: 'input',
-              name: 'endDate',
-              placeholder: 'Enter a end date',
-            }}
-            value={formData.endDate}
-            rules={{}}
-          />
-        </Hide>
+        <Input
+          label="End Date:"
+          type="date"
+          htmlAttrs={{
+            type: 'input',
+            name: 'endDate',
+            placeholder: 'Enter a end date',
+            disabled: formData.isOpenEnded,
+          }}
+          value={formData.endDate}
+          rules={{}}
```

Comment:
Just curious as I don't know the answer, does rules have to be there since it's an empty object? It's expected I assume, just checking :)

Reply:
Not sure as this that prop being passed in was already there. I assume it requires an empty object to full-fill it but will have a look into whether it can be removed